invest-flask.spec

```diff
@@ -48,17 +48,12 @@ invest_pyz = PYZ(invest_a.pure, invest_a.zipped_data, cipher=block_cipher)
 server_pyz = PYZ(server_a.pure, server_a.zipped_data, cipher=block_cipher)
 
 # Create the executable file.
-if is_darwin:
-    # add rtree dependency dynamic libraries from conda environment
-    invest_a.binaries += [
-        (os.path.basename(name), name, 'BINARY') for name in
-        glob.glob(os.path.join(conda_env, 'lib/libspatialindex*.dylib'))]
-elif is_linux:
+if not is_win:
     # add rtree dependency dynamic libraries from conda environment
     invest_a.binaries += [
         (os.path.basename(name), name, 'BINARY') for name in
         glob.glob(os.path.join(conda_env, 'lib/libspatialindex*'))]
-elif is_win:
+else:
```

Comment:
Since we've made the switch to conda for the invest build workflows, could we add:

```python
# add rtree dependency dynamic libraries from conda environment
      invest_a.binaries += [
          (os.path.basename(name), name, 'BINARY') for name in
          glob.glob(os.path.join(conda_env, 'Library/bin/spatialindex*.dll'))]
```

This would also mean a change on line 9,10 to set `conda_env` for windows as well.

Reply:
(Hi, Rich)